### PR TITLE
Issue #3171 Tests fail to get status of container in older PS/cmd

### LINKED ISSUE #3171: Integration tests fail to get status of container on Windows with specific PS/cmd versions
### General information

  * OS: Windows

### Steps to reproduce

  1. run basic.feature on Windows

### Expected

Status should be found.

### Actual

```
  Scenario: OpenShift is ready after startup                                                                                                     # features\basic.feature:49
    Given Minishift has state "Running"                                                                                                          # testsuite.go:77 -> *Minishift
error getting state of container 'k8s_webconsole_webconsole' with ID '.ID' - error: 'Error: No such object: .ID
Command failed: exit status 1
', retrying in 6s...
```

### PR DESCRIPTION
**Pull request by agajdosi**
Fixes #3171 

Steps to test the pull request

  1. `make integration GODOG_OPTS=-tags=basic`

I've tested the PR on Windows 7 and 10 and Fedora, works fine.

### CODE REVIEW

**Comment by agajdosi** [comment]
retest this please

**Comment by agajdosi** [comment]
retest this please

**Comment by agajdosi** [comment]
retest this please

**Comment by agajdosi** [comment]
retest this please

**Comment by agajdosi** [comment]
This PR is ready for review. It is just a small change in formatting to fix problems on some versions of cmd and PS on Windows. Please take a look on it when you have time @LalatenduMohanty @anjannath.

**Comment by agajdosi** [comment]
retest this please

File: test/integration/features/cmd-openshift.feature [hunk 1]
```diff
@@ -36,10 +36,10 @@ cluster in VM provided by Minishift.
         finished last time. When container is new and had never finished then this time value is set to 0001-01-01T00:00:00Z.
         On restart of OpenShift cluster containers are terminated, which sets FinishedAt to actual time. This value persist
         after next start of container.
-    Given stdout of command "minishift ssh -- "docker inspect --format={{.State.FinishedAt}} origin"" is equal to "0001-01-01T00:00:00Z"
+    Given stdout of command "minishift ssh -- 'docker inspect --format={{.State.FinishedAt}} origin'" is equal to "0001-01-01T00:00:00Z"
      When executing "minishift openshift restart" succeeds
      Then stdout should contain "OpenShift restarted successfully"
-      And stdout of command "minishift ssh -- "docker inspect --format={{.State.FinishedAt}} origin"" is not equal to "0001-01-01T00:00:00Z"
+      And stdout of command "minishift ssh -- 'docker inspect --format={{.State.FinishedAt}} origin'" is not equal to "0001-01-01T00:00:00Z"
```
Comment (LalatenduMohanty) [inline test/integration/features/cmd-openshift.feature]: Interesting to see only the `"` was causing the issue.

**Comment by LalatenduMohanty** [comment]
Waiting for CI to finish.

**Pull request merged**